packages/create-turbo/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "create-turbo",
-  "version": "1.8.8",
+  "version": "1.9.0-alpha.0",
```

Comment:
we probably don't want to bump this here?

Reply:
This is technically the "latest" published version, and it will get overridden post merge and deploy of the next version so it wouldn't be breaking, but there's also no harm in removing it either, will revert.